openshift/templates/strapi/bc.yaml

```diff
@@ -59,10 +59,6 @@ objects:
         uri: ${GIT_REPO}
       type: Git
     strategy:
```

Comment:
Why did we remove this?

Reply:
It was throwing an error when the project was building.  I looked into what this block of yaml was doing, turns out it's a relic from when the bc.yaml was generated. The fact that our DockerFile was already importing the "node:12.17.0-alpine" meant it was redundant.
Not sure why it was erroring in OCP4 and not OCP3, but I double checked with Patrick once it was deploying and  he said there were no issues with the deletion.